src/app/api/auth/[...nextauth]/route.tsx

```diff
@@ -0,0 +1,25 @@
+import NextAuth from 'next-auth';
+import GoogleProvider from "next-auth/providers/google";
+import { SessionOptions } from 'next-auth';
+
+
+const googleClientId = process.env.GOOGLE_ID;
+const googleClientSecret = process.env.GOOGLE_SECRET;
+
+if (!googleClientId || !googleClientSecret) {
+    console.log("sad");
```

Comment:
hate to be no fun.. but could you pls change this to smth like "missing google auth secrets in env"